src/components/scenes/WalletListScene.js

```diff
@@ -83,21 +86,41 @@ class WalletListComponent extends React.PureComponent<Props, State> {
     }
   }
 
-  componentDidMount() {
-    if (this.props.needsPasswordCheck) {
-      Airship.show(bridge => <PasswordReminderModal bridge={bridge} />)
-    } else {
-      this.showTutorial()
+  getSortCache = async () => {
```

Comment:
This is a waste of another file to synchronize, just put this in the Settings object and read it from redux

Reply:
edge-react-gui/src/modules/Core/Account/settings.js

Reply:
Big changes, fixed